<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPDConfDialog</class>
 <widget class="QDialog" name="MPDConfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPD Server Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MPD Server Configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host name, IP address or Unix socket of the MPD Server:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="hostLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="portLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="dirLabel">
     <property name="toolTip">
      <string>If the MPD server is not local (not using a Unix socket), the Music directory cannot be determined automatically.</string>
     </property>
     <property name="text">
      <string>MPD Music Directory (For &quot;remote&quot; servers)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="dirLineEdit">
     <property name="toolTip">
      <string>If the MPD server is not local (not using a Unix socket), the Music directory cannot be determined automatically.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MPDConfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MPDConfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
